src/test/java/com/woowacourse/acceptance/RestaurantAcceptanceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.woowacourse.acceptance;
+
+import static com.woowacourse.acceptance.support.RestAssuredRequest.httpGetRequest;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+class RestaurantAcceptanceTest extends AcceptanceTest {
```

Comment:
```suggestion
public class RestaurantAcceptanceTest extends AcceptanceTest {
```
저도 지금 실수했을게 많은 부분이긴합니다... 제꺼도 찾아주세요 ㅋ_ㅋ